src/content/guides/tree-shaking.md

```diff
@@ -198,6 +198,7 @@ With that squared away, we can run another `npm run build` and see if anything h
 
 Notice anything different about `dist/bundle.js`? Clearly the whole bundle is now minified and mangled, but, if you look carefully, you won't see the `square` function included but will see a mangled version of the `cube` function (`function r(e){return e*e*e}n.a=r`). With minification and tree shaking our bundle is now a few bytes smaller! While that may not seem like much in this contrived example, tree shaking can yield a significant decrease in bundle size when working on larger applications with complex dependency trees.
 
+T> [ModuleConcatenationPlugin](https://webpack.js.org/plugins/module-concatenation-plugin/) is needed for the tree shaking to work. It is added by `mode: "production"`. If you are not using it, remember to add the [ModuleConcatenationPlugin](https://webpack.js.org/plugins/module-concatenation-plugin/) manually.
```

Comment:
It is not necessary to use absolute URLs, relative URLs are fine: `/plugins/module-concatenation-plugin`